src/commands/widget-sourcemaps.command.ts

```diff
@@ -0,0 +1,14 @@
+import { ContentService } from "../services/content.service";
+import { WidgetSourcemapsManagerFactory } from "../content/factory/widget-sourcemaps-manager.factory";
+
+export class WidgetSourcemapsCommand {
+    private contentService = new ContentService();
+    private widgetManagerFactory = new WidgetSourcemapsManagerFactory();
```

Comment:
This should be named `widgetSourceMapFactory`.

Reply:
60e772bfccc6eca503e06dc2e9dc0d10041f05d1